packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -212,6 +212,12 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 				this.detachedRevision,
 			);
 		}
+
+		// Don't submit the op if it is not attached
```

Comment:
No objections to this change because encoding the message is some nontrivial amount of work which we throw away, but I don't think the root cause of the bug reported in the attached work item is understood. In fact, `SharedObjectCore` already should skip submitting ops: https://github.com/microsoft/FluidFramework/blob/8cf4ad54c6df4f7746b4ddee5e933a7ff7c166d4/packages/dds/shared-object-base/src/sharedObject.ts#L357

Most DDSes have explicit checks to either avoid doing extra work they know will be thrown away or because it matters to them whether they're attached (since the resulting pending state will be different as when detached they expect no ack of their operation)